### DISCUSSION TITLE
Task4 奖励发放🎉

### DISCUSSION THREAD

**Discussion by phoouze** [openbuildxyz/mina-zkapp-bootcamp#474] [Announcements]
[Task4](https://github.com/openbuildxyz/mina-zkapp-bootcamp/blob/main/task/task4.md) 的奖励已发放，请查看[奖励统计表](https://github.com/openbuildxyz/mina-zkapp-bootcamp/blob/main/reward/task4.md)🎉

没有收到奖励的同学请：

查看自己的 pr 是否被合并，有问题的 pr 请及时处理。
查看自己是否在 learn/username/username.md 中提交钱包地址，未提交钱包地址的请尽快提交。
是否按照[规则](https://github.com/openbuildxyz/mina-zkapp-bootcamp#%E4%BB%BB%E5%8A%A1%E6%8F%90%E4%BA%A4)提交。
只有在 Task4 任务时间内(12月16日前)提交才有奖励， 请确认。
请检查自己的提交内容和 pr，及时更新。漏发的同学会进行补发，有任何问题请在本讨论区进行讨论。

**Comment by xtheing** [comment]
大佬，我task3的代码还没有合并进去啊 [421](https://github.com/openbuildxyz/mina-zkapp-bootcamp/pull/421#issue-2732955042)，task4 和 task5的代码也都完成了，是不能一起提交吗？坑也占了啊，车已经开走了吗？

**Reply by phoouze** [reply]
已经合并了，之前占过坑， 继续提交task4和task5都有奖励的

**Reply by xtheing** [reply]
@smallfu6 task4 和 task5 https://github.com/openbuildxyz/mina-zkapp-bootcamp/pull/484 没补到啊，大佬

**Reply by phoouze** [reply]
@xtheing 发了

**Reply by xtheing** [reply]
感谢大佬的猪脚饭
